tools/integration_tests/emulator_tests/write_stalls/write_stall_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package emulator_tests
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/mounting/static_mounting"
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/setup"
+)
+
+const port = 8020
+
+var (
+	testDirPath string
+	mountFunc   func([]string) error
+	// root directory is the directory to be unmounted.
+	rootDir string
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Can we make TestMain() common for all the emulator based test?

And then a file like `write_stall_test.go` for different test suites. So, if we need to add another test tomorrow then we will just implement `read_stall_test.go`. That's it.

If it's possible then we can remove this intermediate folder `write_stalls`.